gitlab/v4/objects/merge_requests.py

```diff
@@ -409,6 +411,16 @@ class ProjectMergeRequestManager(CRUDMixin, RESTManager):
     }
 
 
+class ProjectDeploymentMergeRequest(ProjectMergeRequest):
+    pass
+
+
+class ProjectDeploymentMergeRequestManager(ListMixin, RESTManager):
+    _path = "/projects/%(project_id)s/deployments/%(deployment_id)s/merge_requests"
+    _obj_cls = ProjectDeploymentMergeRequest
+    _from_parent_attrs = {"deployment_id": "id", "project_id": "project_id"}
+
```

Comment:
https://docs.gitlab.com/ee/api/deployments.html#list-of-merge-requests-associated-with-a-deployment

> It supports the same parameters as the Merge Requests API and returns a response using the same format: 

So should this class be derived from a MergeRequest class?

Reply:
Makes sense, changed. CLI when inheriting from MergeRequestManager:
```shell
$ gitlab project-deployment-merge-request list --help
usage: gitlab project-deployment-merge-request list [-h] [--sudo SUDO] --deployment-id DEPLOYMENT_ID --project-id PROJECT_ID [--state STATE] [--order-by ORDER_BY]
                                                    [--sort SORT] [--milestone MILESTONE] [--view VIEW] [--labels LABELS] [--with-labels-details WITH_LABELS_DETAILS]
                                                    [--with-merge-status-recheck WITH_MERGE_STATUS_RECHECK] [--created-after CREATED_AFTER] [--created-before CREATED_BEFORE]
                                                    [--updated-after UPDATED_AFTER] [--updated-before UPDATED_BEFORE] [--scope SCOPE] [--author-id AUTHOR_ID]
                                                    [--author-username AUTHOR_USERNAME] [--assignee-id ASSIGNEE_ID] [--approver-ids APPROVER_IDS]
                                                    [--approved-by-ids APPROVED_BY_IDS] [--reviewer-id REVIEWER_ID] [--reviewer-username REVIEWER_USERNAME]
                                                    [--my-reaction-emoji MY_REACTION_EMOJI] [--source-branch SOURCE_BRANCH] [--target-branch TARGET_BRANCH] [--search SEARCH]
                                                    [--in IN] [--wip WIP] [--not NOT] [--environment ENVIRONMENT] [--deployed-before DEPLOYED_BEFORE]
                                                    [--deployed-after DEPLOYED_AFTER] [--page PAGE] [--per-page PER_PAGE] [--all]
```